.github/workflows/tests.yml

```diff
@@ -1,24 +1,18 @@
 # A single CI script with github workflow to test NNPDF, and upload the conda package and documentation
-name: Tests
+name: Test conda package
 
 on: [push]
 
 concurrency:
   group: ${{ github.head_ref || github.ref }}
   cancel-in-progress: true
 
-
 jobs:
   build:
     strategy:
       matrix:
-        os: [ubuntu-latest] # , macos-latest]
-        python-version: ["3.11"] # ["3.9", "3.10", "3.11"]
-        include:
-          - os: ubuntu-latest
-            CONDA_OS: linux-64
-          - os: macos-latest
-            CONDA_OS: osx-64
+        os: [ubuntu-latest, macos-latest]
+        python-version: ["3.9", "3.11"]
```

Comment:
```suggestion
        python-version: ["3.9", "3.11"] # only test for the outer values for the range we support to prevent the cluster from refusing connections due to too many requests
```